src/clp_ffi_js/ir/StructuredIrStreamReader.cpp

```diff
@@ -26,6 +27,20 @@ namespace {
 constexpr std::string_view cEmptyJsonStr{"{}"};
 constexpr std::string_view cReaderOptionsLogLevelKey{"logLevelKey"};
 constexpr std::string_view cReaderOptionsTimestampKey{"timestampKey"};
+
+/**
+ * @see nlohmann::basic_json::dump
+ * Dumps a JSON value with error handling set to replace invalid UTF-8 characters rather than
```

Comment:
Would it be more clear if we say
"Serializes a JSON value into a string with invalid UTF-8 sequences replaced rather than ..."